docs/source/notes/modules.rst

```diff
@@ -603,10 +603,12 @@ PyTorch provides two types of hooks for modules:
   :func:`~torch.nn.modules.module.register_module_forward_pre_hook` and
   :func:`~torch.nn.modules.module.register_module_forward_hook` functions.
 * **Backward hooks** are called during the backward pass. They can be installed with
-  :func:`~torch.nn.Module.register_full_backward_hook`. These hooks will be called when the backward for this
-  Module has been computed and will allow the user to access the gradients for both the inputs and outputs.
+  :func:`~torch.nn.Module.register_backward_pre_hook` and :func:`~torch.nn.Module.register_full_backward_hook`.
```

Comment:
can we keep the "full" in the name.
It is a bit weird but that improves consistency.

Reply:
hmm I'm curious what the "full" means in the name

Reply:
Nothing. We just needed a different name than the current `register_backward_hook()` when the new version was added.
One day maybe we'll deprecate the old ones that are completely broken and use the full ones by default :dream: